.github/CODEOWNERS

```diff
@@ -1,7 +1,29 @@
-* @microsoft/prompt-flow-approvers
+# CAUTION: Order is important; the LAST matching pattern takes the most precedence.
+# So put outside folders first, and then inside folders.
+# Don't change the sequence of the regions.
 
-/src/promptflow-tracing/promptflow/tracing/_start_trace.py @microsoft/promptflow-sdk
+* @microsoft/prompt-flow-approver
+
+# 1. Folders
+/src/promptflow-tracing/ @microsoft/promptflow-execution
+
+/src/promptflow-eval/ @microsoft/pf-eval-team-contributor
+
+/src/promptflow-core/ @microsoft/promptflow-sdk
+/src/promptflow-devkit/ @microsoft/promptflow-sdk
+/src/promptflow-azure/ @microsoft/promptflow-sdk
+/src/promptflow-recording/ @microsoft/promptflow-sdk
+/src/promptflow-execution/ @microsoft/promptflow-sdk
```

Comment:
I think we don't have this folder? promptflow-execution

Reply:
GPT auto generation....